.github/workflows/ci.yml

```
@@ -1,7 +1,7 @@
 name: CI
-on:
-  - push
-  - pull_request
+# on:
+#   - push
+#   - pull_request
```

Feel free to enable the ci by uncommenting these
```suggestion
on:
  - push
  - pull_request
```